util/crc32c.cc

```diff
@@ -18,9 +18,32 @@
 #endif
 #include "util/coding.h"
 
+#ifdef __powerpc64__
+#include "util/crc32c_ppc.h"
+#include "util/crc32c_ppc_constants.h"
+
+#if __linux__
+#include <sys/auxv.h>
+
+#ifndef PPC_FEATURE2_VEC_CRYPTO
+#define PPC_FEATURE2_VEC_CRYPTO 0x02000000
```

Comment:
These symbols are present in the header files just above.  During an earlier round of review Daniel Axtens had me add these as a backup in case the headers are not found.  For example, on powerpc64 freebsd (which doesn't have all the same headers as linux) we hit compilation errors in ceph and had to fix things like this to be more careful.